docs/standard/frameworks.md

```diff
@@ -73,6 +73,7 @@ The following table shows the compatibility of the .NET 5+ TFMs.
 | net7.0-ios         | (Subsequent version of `net6.0-ios`)                                                                                    |
 | net7.0-maccatalyst | (Subsequent version of `net6.0-maccatalyst`)                                                                            |
 | net7.0-macos       | (Subsequent version of `net6.0-macos`)                                                                                  |
+| net7.0-tizen       | TBD (plus everything else inherited from `net7.0`) |
```

Comment:
Need some help on this one.

Reply:
I think as is works. @terrajobst may have the most clarity here.